packages/databricks-vscode/src/configuration/ConnectionManager.ts

```diff
@@ -211,10 +236,12 @@ export class ConnectionManager implements Disposable {
                     this._workspaceClient
                 );
 
-                await this.createWsFsRootDirectory(this._workspaceClient);
                 await this.updateSyncDestinationMapper();
                 await this.updateClusterManager();
-                await this.configModel.set("authParams", authProvider.toJSON());
+                await this.configModel.set(
+                    "authProfile",
+                    authProvider.toJSON().profile as string | undefined
+                );
```

Comment:
If we always save it to the override here, then potential profile changes in the bundle.yml won't take any effect, or do I misunderstand the logic here?

Should we only save the profile to overrides when it doesn't exist in the bundle config? Although I'm not sure if we even currently loading the profile name from the bundle models